hack/hook-me.sh

```diff
@@ -256,122 +256,144 @@ keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+    echo "========================================================================================================================================================"
 
-  echo ""
+    echo ""
 
-  echo "===================================PRE-REQs========================================="
-  echo "\`host\` commands for DNS"
-  echo "\`docker\` https://www.docker.com"
-  echo "===================================================================================="
+    echo "===================================PRE-REQs========================================="
+    echo "\`host\` commands for DNS"
+    echo "\`docker\` https://www.docker.com"
+    echo "\`yq\` https://github.com/mikefarah/yq"
+    echo "===================================================================================="
 
-  echo ""
+    echo ""
 
-  echo "========================================================USAGE======================================================================"
-  echo "> ./hack/hook-me.sh <service e.g., gardener-extension-provider-aws> <extension namespace e.g. extension-provider-aws-fpr6w> <webhookserver port e.g., 8443> [<quic-server port, e.g. 9443>]"
-  echo "> \`make EXTENSION_NAMESPACE=<extension namespace e.g. extension-provider-aws-fpr6w> WEBHOOK_CONFIG_MODE=service start\`"
-  echo "=================================================================================================================================="
+    echo "========================================================USAGE======================================================================"
+    echo "> ./hack/hook-me.sh <service e.g., gardener-extension-provider-aws> <extension namespace e.g. extension-provider-aws-fpr6w> <webhookserver port e.g., 8443> [<quic-server port, e.g. 9443>]"
+    echo "> \`make EXTENSION_NAMESPACE=<extension namespace e.g. extension-provider-aws-fpr6w> WEBHOOK_CONFIG_MODE=service start\`"
+    echo "=================================================================================================================================="
 
-  echo ""
+    echo ""
 
-  echo "===================================CLEAN UP COMMANDS========================================="
-  echo ">  kubectl -n $namespace delete  svc/quic-lb"
-  echo ">  kubectl -n $namespace delete  deploy/quic-server"
-  echo "============================================================================================="
+    echo "===================================CLEAN UP COMMANDS========================================="
+    echo "> kubectl -n $namespace delete --ignore-not-found svc/quic-lb"
+    echo "> kubectl -n $namespace delete --ignore-not-found deploy/quic-server"
+    echo "============================================================================================="
 
-  exit 0
+    exit 0
 }
 if [[ "${BASH_SOURCE[0]}" = "$0" ]]; then
 
-  if [ "$1" == "-h" ] ; then
+    if [ "$1" == "-h" ] ; then
```

Comment:
```suggestion
    if [ "$#" -lt 2 ] || [ "$1" == "-h" ]; then
```

Otherwise if we call the script without any arguments the following error is displayed: `hack/hook-me.sh: line 363: $1: unbound variable`